scripts/release-build.js

```diff
@@ -15,18 +15,23 @@ function createDataBundle() {
 }
 
 // Returns a promise for writing the data to JSON file
-async function writeData() {
+async function writeData(data) {
   const dest = path.resolve(directory, 'data.json');
-  const data = createDataBundle();
   await fs.writeFile(dest, data);
 }
 
-async function writeIndex() {
-  const dest = path.resolve(directory, 'index.js');
+async function writeIndexCJS() {
+  const dest = path.resolve(directory, 'index.cjs');
   const content = `module.exports = require("./data.json");\n`;
```

Comment:
This function was to keep compatibility with ESM where JSON couldn't be directly imported (#10033). Seems like it's not needed here anymore.

Reply:
Makes sense. I changed package.json to the following:
```json
{
    "main": "data.json",
    "exports": { "import": "./index.js", "require": "./data.json" },
}
```